build.sbt

```diff
@@ -1,4 +1,5 @@
 import com.typesafe.tools.mima.core._
+import explicitdeps.ExplicitDepsPlugin.autoImport.moduleFilterRemoveValue
```

Comment:
Is this due to https://github.com/cb372/sbt-explicit-dependencies/issues/33?

Reply:
Yes.